graphql2/graphqlapp/destinationdisplayinfo.go

```diff
@@ -21,40 +22,15 @@ type (
 func (a *App) Destination() graphql2.DestinationResolver           { return (*Destination)(a) }
 func (a *App) DestinationInput() graphql2.DestinationInputResolver { return (*DestinationInput)(a) }
 
-func (a *DestinationInput) Args(ctx context.Context, obj *graphql2.DestinationInput, args map[string]string) error {
-	obj.Args = args
-
-	obj.Values = make([]graphql2.FieldValueInput, 0, len(args))
-	for k, v := range args {
-		obj.Values = append(obj.Values, graphql2.FieldValueInput{FieldID: k, Value: v})
-	}
-
-	return nil
-}
-
-func (a *DestinationInput) Values(ctx context.Context, obj *graphql2.DestinationInput, values []graphql2.FieldValueInput) error {
-	obj.Values = values
+func (a *DestinationInput) Values(ctx context.Context, obj *gadb.DestV1, values []graphql2.FieldValueInput) error {
 	obj.Args = make(map[string]string, len(values))
 	for _, val := range values {
 		obj.Args[val.FieldID] = val.Value
 	}
 	return nil
 }
 
-func (a *Destination) Args(ctx context.Context, obj *graphql2.Destination) (map[string]string, error) {
-	if obj.Args != nil {
-		return obj.Args, nil
-	}
-
-	m := make(map[string]string, len(obj.Values))
-	for _, val := range obj.Values {
-		m[val.FieldID] = val.Value
-	}
-
-	return m, nil
-}
-
-func (a *Destination) Values(ctx context.Context, obj *graphql2.Destination) ([]graphql2.FieldValuePair, error) {
+func (a *Destination) Values(ctx context.Context, obj *gadb.DestV1) ([]graphql2.FieldValuePair, error) {
```

Comment:
I don't see any error checks here, perhaps we can set the return type to only `[]graphql2.FieldValuePair`?

Reply:
This is a GraphQL resolver (i.e., compatibility resolver), so it needs to have that specific signature (args and return values) even if it doesn't have an error condition.